roles/confluent.ksql/templates/ksql-server_log4j.properties.j2

```diff
@@ -45,12 +45,66 @@ log4j.appender.kafkaAppender.MaxFileSize={{ksql.kafka_appender_log_file_size}}
 log4j.appender.kafkaAppender.MaxBackupIndex={{ksql.kafka_appender_max_log_files}}
 log4j.appender.kafkaAppender.append=true
 
+{% if ksql_log_streaming_enabled | bool %}
+# Log Streaming Configuration
 log4j.appender.kafka_appender=org.apache.kafka.log4jappender.KafkaLog4jAppender
 log4j.appender.kafka_appender.layout=io.confluent.common.logging.log4j.StructuredJsonLayout
-log4j.appender.kafka_appender.BrokerList=localhost:9092
-log4j.appender.kafka_appender.Topic=default_ksql_processing_log
+log4j.appender.kafka_appender.BrokerList={% for host in groups['kafka_broker'] %}{% if loop.index > 1%},{% endif %}{{ host }}:{{kafka_broker_listeners[kafka_broker_inter_broker_listener_name]['port']}}{% endfor %}
+log4j.appender.kafka_appender.Topic={{ksql_processing_log}}
 log4j.logger.processing=ERROR, kafka_appender
 
+# Security Configuration for Log Streaming
+{% set listener = kafka_broker_listeners[kafka_broker_inter_broker_listener_name] %}
+{% set truststore_path = ksql_truststore_path %}
+{% set truststore_storepass = ksql_truststore_storepass %}
+{% set keystore_path = ksql_keystore_path %}
+{% set keystore_storepass = ksql_keystore_storepass %}
+{% set keystore_keypass = ksql_keystore_keypass %}
+{% set sasl_plain_username = sasl_plain_users.ksql.principal %}
+{% set sasl_plain_password = sasl_plain_users.ksql.password %}
+{% set sasl_scram_username = sasl_scram_users.ksql.principal %}
+{% set sasl_scram_password = sasl_scram_users.ksql.password %}
+{% set kerberos_keytab_path = ksql_kerberos_keytab_path %}
+{% set kerberos_principal = ksql_kerberos_principal %}
+{% set oauth_username = ksql_ldap_user %}
+{% set oauth_password = ksql_ldap_password %}
```

Comment:
are these set lines necessary? you could use what the variables are set to below.

i would keep the 
`{% set listener = kafka_broker_listeners[kafka_broker_inter_broker_listener_name] %}`
line, it improves readablility below

Reply:
Fair.